### PR TITLE
Fix #583 VR not displayed without change LUT

### LINKED ISSUE #583: [Tests] 3.1.1 and master- Voi Cutter in VR not displayed
How to:
 * open a mask in Voi Cutter (Segmentation workspace)
 * the mask automatically switch to 3D -> it is not displayed in VR mode, except if you change the Lut

### PR DESCRIPTION
**Pull request by Florent2305**

### CODE REVIEW

File: src/layers/legacy/medVtkInria/vtkImageView/vtkImageView3D.cxx [hunk 1]
```diff
@@ -1287,3 +1289,22 @@ medVtkImageInfo* vtkImageView3D::GetMedVtkImageInfo(int layer /*= 0*/) const
 
     return imageInfo;
 }
+
+
```
Comment (mathildemerle) [inline src/layers/legacy/medVtkInria/vtkImageView/vtkImageView3D.cxx:1293]: you can remove this line

File: src/layers/legacy/medVtkInria/vtkImageView/vtkImageView3D.cxx [hunk 2]
```diff
@@ -1287,3 +1289,22 @@ medVtkImageInfo* vtkImageView3D::GetMedVtkImageInfo(int layer /*= 0*/) const
...
+void  vtkImageView3D::initImage(int pi_iLayer)
```
Comment (mathildemerle) [inline src/layers/legacy/medVtkInria/vtkImageView/vtkImageView3D.cxx:1294]: is it possible to give it a clearer name? Something like "initializeTransferFunctions"

File: src/layers/legacy/medVtkInria/vtkImageView/vtkImageView3D.cxx [hunk 3]
```diff
@@ -1287,3 +1289,22 @@ medVtkImageInfo* vtkImageView3D::GetMedVtkImageInfo(int layer /*= 0*/) const
...
+{
+    if (pi_iLayer == 0)
+    {
+        double *range = m_poInternalImageFromInput->GetScalarRange();
```
Comment (mathildemerle) [inline src/layers/legacy/medVtkInria/vtkImageView/vtkImageView3D.cxx:1298]: is it possible to move these 2 lines in SetFirstLayer? It would be clear

Comment (Florent2305) [reply src/layers/legacy/medVtkInria/vtkImageView/vtkImageView3D.cxx:1298]: It's really make sens to stay in the initialization of TransferFunctions phase.

File: src/layers/legacy/medVtkInria/vtkImageView/vtkImageView3D.cxx (hunk 2 above)
Comment (Florent2305) [reply src/layers/legacy/medVtkInria/vtkImageView/vtkImageView3D.cxx:1294]: ok

File: src/layers/legacy/medVtkInria/vtkImageView/vtkImageView3D.cxx (hunk 1 above)
Comment (Florent2305) [reply src/layers/legacy/medVtkInria/vtkImageView/vtkImageView3D.cxx:1293]: Je ne suis vraiment pas fan des fichiers qui se finisses brutalement sur une accolade, mais bon si tu veux.

**Pull request merged**